src/uct/ib/base/ib_md.c

```diff
@@ -1360,10 +1360,21 @@ static ucs_status_t uct_ib_verbs_md_open(struct ibv_device *ibv_device,
         goto err_md_free;
     }
 
-    if (IBV_DEVICE_ATOMIC_HCA(dev)) {
+    if (IBV_DEVICE_ATOMIC_HCA(dev) ||
+        IBV_DEVICE_ATOMIC_GLOB(dev)) {
         dev->atomic_arg_sizes = sizeof(uint64_t);
     }
 
+    if (IBV_DEVICE_ATOMIC_GLOB(dev)) {
+        dev->pci_fadd_arg_sizes = sizeof(uint64_t);
+        dev->pci_cswap_arg_sizes = sizeof(uint64_t);
+    }
+
+    if (IBV_DEVICE_ATOMIC_GLOB(dev)) {
+        dev->pci_fadd_arg_sizes = sizeof(uint64_t);
+        dev->pci_cswap_arg_sizes = sizeof(uint64_t);
+    }
```

Comment:
seems these lines are duplicated?

Reply:
i.e duplicating lines 1368-1371

Reply:
Yes. This is by mistake. Will correct it

Reply:
Corrected. Please review